.travis.yml

```diff
@@ -73,7 +73,15 @@ script:
       make -C docs clean
       python setup.py docs
     fi
+    source ./rultor_secrets.sh
   - bash .misc/deploy.coverage.sh
 
+deploy:
+  provider: pypi
+  server: https://testpypi.python.org/pypi
+  user: "$PYPIUSER"
+  password: "$PYPIPW"
+  distributions: "bdist_wheel" # Your distributions here
```

Comment:
Hold up there, I don't think we should be deploying from travis without discussion. Right now we use rultor which provides an easy way to provide secrets. We might actually want you to modify rultor instead.